minari/data_collector/data_collector.py

```diff
@@ -361,7 +376,7 @@ def clear_buffer(dictionary_buffer: EpisodeBuffer, episode_group: h5py.Group):
                             episode_group.create_dataset(key, data=np_data, chunks=True)
 
         for i, eps_buff in enumerate(self._buffer):
-            if len(eps_buff["actions"]) == 0:
+            if "actions" not in eps_buff:
```

Comment:
what does this line do exactly? might be good to comment

Reply:
I added a comment

Reply:
Much cleaner now, good stuff